ui/components/component-library/tag/tag.scss

```diff
@@ -0,0 +1,7 @@
+.tag {
+  height: auto;
+  width: max-content;
+  padding-top: 1px;
+  padding-bottom: 1px;
+  border-radius: 9999px;
```

Comment:
Now that the borderRadius={BORDER_RADIUS.PILL} option is available I think we can replace this

Reply:
Done